### PR TITLE
Network

### PR DESCRIPTION
**Pull request by Moussiaa**

### CODE REVIEW

File: src/main/java/EarthquakeService.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package adry.earthquake;
```
[the remark this answers is not in the document]
Comment (Moussiaa) [reply src/main/java/EarthquakeService.java]: How does one go about fixing such an error

[the remark this answers is not in the document]
Comment (andrewoid) [reply src/main/java/EarthquakeService.java]: Move it to the correct directory

File: src/main/java/adry/earthquake/EarthquakeFrame.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        setSize(300, 600);
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        setLayout(new BorderLayout());
+        JRadioButton hour;
+        JRadioButton month;
+
+        hour = new JRadioButton("One Hour");
+        hour.addActionListener(e -> {
+            adry.earthquake.EarthquakeService service = new EarthquakeServiceFactory().getService();
```
Comment (andrewoid) [inline src/main/java/adry/earthquake/EarthquakeFrame.java]: Create this only once.

File: src/main/java/adry/earthquake/EarthquakeFrame.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+    private void handleResponse2(FeatureCollection response) {
+
+        String[] listData1 = new String[response.features.length];
+        for (int i = 0; i < response.features.length; i++) {
+            Feature feature = response.features[i];
+            listData1[i] = feature.properties.mag + " " + feature.properties.place;
+        }
+        jlist.setListData(listData1);
+    }
```
Comment (andrewoid) [inline src/main/java/adry/earthquake/EarthquakeFrame.java]: Aren't these two methods exactly the same?

File: src/main/java/adry/earthquake/json/Feature.java [hunk 4]
```diff
@@ -0,0 +1,10 @@
+package adry.earthquake.json;
+
+public class Feature {
+
+    public Properties properties = new Properties();
+    double mag;
+    String place;
+    long time;
```
Comment (andrewoid) [inline src/main/java/adry/earthquake/json/Feature.java]: These are not in Feature

File: src/main/java/adry/earthquake/EarthquakeFrame.java (hunk 2 above)
Comment (Moussiaa) [reply src/main/java/adry/earthquake/EarthquakeFrame.java]: make one action listener that can be applied to both buttons?

Comment (andrewoid) [reply src/main/java/adry/earthquake/EarthquakeFrame.java]: No. Create only one EarthquakeService

File: src/main/java/adry/earthquake/EarthquakeService.java [hunk 5]
```diff
@@ -0,0 +1,15 @@
+package adry.earthquake;
+
+import adry.earthquake.json.FeatureCollection;
+import io.reactivex.rxjava3.core.Single;
+import retrofit2.http.GET;
+
+public interface EarthquakeService {
+
+    @GET ("/earthquakes/feed/v1.0/summary/1.0_hour.geojson")
+    public abstract Single<FeatureCollection> oneHour();
```
Comment (andrewoid) [inline src/main/java/adry/earthquake/EarthquakeService.java]: Remove "public abstract"

**Comment by andrewoid** [comment]
Add a Readme and screenshots

File: README.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+### Earthquake 
+
+
+
+![screenshot when it works] (![1hour.png](..%2F..%2FDesktop%2F1hour.png))
```
Comment (andrewoid) [inline README.md]: The location of the image doesn't work

Comment (Moussiaa) [reply README.md]: It's still not working after clearing the caches so I put nothing there till I can get a screenshot

Comment (andrewoid) [reply README.md]: Let's meet a few minutes before class.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**